### ISSUE TITLE
PhatLoots not working with Spigot 1.10?

### ISSUE THREAD

**Issue by WesleyDW** [Codisimus/PhatLoots#54]
Hey!

Whenever I am using the /loot command, the only thing i get is "/loot help". But whatever I do, it just doesn't work. if, for example, i use /loot add dirt, it prints ''/loot help'' again.

I am using version 4.2.3 with the newest spigot build. What should I do?

**Comment by tara81** [comment]
Wait for cody to come back. Author missing for 1 month now.  That is really sad to hear.  I didnt update yet.

**Comment by Chevels** [comment]
Indeed! My server is only waiting for this plugin to go to 1.10. All our RP depends on that plugin, so we're looking forward to having a compatible 1.10 version!

Anyway thank you for your excellent work Codissimus!

**Comment by tara81** [comment]
i got a tweet from him 1 month ago saying he was sorry for the inactivity,
in the process of moving. But never said anything about when he would be
back.

**Comment by Codisimus** [comment]
Hello and thank you for your patience. I will be looking at what is need to get this up to date after work today.

**Comment by tara81** [comment]
Im so happy you are back codi, can you help with the network booster after you update?

**Comment by Codisimus** [comment]
Still working on getting the latest Spigot build.

@tara81 I haven't gone through all of my emails/tickets yet so I'm not sure what you mean by the network booster. If we talked about it before I must've forgotten what it was.

**Comment by Chevels** [comment]
This sound good! Thank you for the work :)

**Comment by tara81** [comment]
https://github.com/Codisimus/PhatLoots/issues/36  network booster  - mojang has started enforcing eula, it is very hard to sell perks now on servers.  Mojang contacted me as well to change my store.  I am willing to pay you $20 usd to add this feature to 1.9.4 and 1.10.2 (I have to stay on 1.9.4 til rpg inventory plugin and yours gets updated).

**Comment by Codisimus** [comment]
Ok, I have an update. Not sure if this is a good thing or bad but I downloaded version 4.2.3 from my website, built the latest version of Spigot (1.10.2), threw them on a fresh server and everything works fine.

**Comment by Codisimus** [comment]
Any more news on this? I recently hosted a Minecraft event using 1.10.2 and everything worked fine. I am thinking maybe a plugin conflict?

**Comment by Chevels** [comment]
I dare not move my server on 1.10^^. I was waiting for confirmation that Phatloots (and Logblocks!) Works properly. I can not take the risk of trying without knowing. 
I can not take the risk of trying without knowing. But if it works for you it reassures me. I would do on my return because I travel 3 weeks

**Comment by tara81** [comment]
Codi can you add network booster to the plugin features?> Mojang is giving me 2 weeks to change my store, And i could really use it. I am offering you $40 usd to add this small feature.

**Comment by Codisimus** [comment]
@tara81 I fixed my CI script today so I can begin committing some code tomorrow. The CI automatically releases dev builds for me. Watch for some updates tomorrow night.

**Comment by tara81** [comment]
that is great news cody ! Great news !
